### DISCUSSION TITLE
Service provider for OpenDataCam wanted

### DISCUSSION THREAD

**Discussion by angieblock** [opendatacam/opendatacam#550] [Q&A]
We are looking for a (paid) service provider who can setup a OpenDataCam + Nano in NRW, Germany for recording traffic data. Please get in touch if you have the required knowledge.

**Comment by shams3049** [comment]
Hi @angieblock Not sure about paid at this point, but I work with a research institute GEMIT from HS Niederrhein. If you like we can set up a meeting, to see how we can help. I have used opendatacam for Nuess and Krefeld Hafeen in NRW.

**Comment by vsaw** [comment]
Hi @angieblock 

Im Valentin, the maintainer of OpenDataCam on GitHub. 

I offer paid consultancy, and development services for OpenDataCam. Feel free to drop me a note on valentin@nono.cam about your needs. 

Best,
Valentin

**Comment by jbaader** [comment]
Hey @angieblock 

I am working for a company based in Germany using a traffic counting system based on OpenDataCam for recording traffic data. We currently have 20 of these devices in use and offer them as a service. 

We can also share the know-how we have gained from setting up our devices and can help you develop your desired solution.

For any inquiries please message: r.pessier@mobilitaetswerk.de

Kind Regards,
Julian

**Comment by vsaw** [comment]
FYI: I converted this into a discussion as issues are being used for only tracking the future development of OpenDataCam.

**Comment by nishantsah08** [comment]
I can do it for you. 
Lets connect  hypertech@outlook.in
